src/translator/translate.py

```diff
@@ -1,22 +1,34 @@
+import logging
+import string
 from typing import List
+
 import six
-import logging
+from cpr_sdk.parser_models import ParserOutput
+from google.cloud import translate_v2
 from tenacity import retry, stop_after_attempt, wait_random_exponential
 
+_LOGGER = logging.getLogger(__file__)
+
 
-from cpr_sdk.parser_models import ParserOutput
-from google.cloud import translate_v2
+def should_translate_text(text: str) -> bool:
+    """
+    Identify whether we should translate text.
 
+    For example punctuation and numbers shouldn't be translated as they are the same in
+    most languages.
+    """
+    if all(char in string.punctuation or char.isdigit() for char in text):
+        return False
```

Comment:
this doesn't account for spaces too. I think it'd be worth updating to include that?
```suggestion
    if all(char in string.punctuation or char.isdigit() or char.isspace() for char in text):
        return False
```